Model/src/GamePackage/Player.java

```diff
@@ -1,4 +1,265 @@
 package GamePackage;
 
+import java.util.ArrayList;
+
+import BoardPackage.BoardSquare;
+import BoardPackage.PropertySquare;
+import CardPackage.Card;
+
+/**********************************************************************
+ * The Player class will keep track of each player's money, properties,
+ * and other stats.
+ * 
+ * @author Dustin Ecker
+ * @version 2/11/2018
+ *********************************************************************/
 public class Player {
+	
+	/** Player's name */
+	private String displayName;
+	
+	/** Player's token */
+	private String token;
+	
+	/** Tells if player can move */
+	private boolean canMove;
+	
+	/** Tells if player is bankrupt */
+	private boolean isBankrupt;
+	
+	/** Holds player's current wallet balance */
+	private int wallet;
+	
+	/** ArrayList that holds player's owned properties */
+	private ArrayList<BoardSquare> propertiesOwned;
+	
+	/** ArrayList that holds player's cards held */
+	private ArrayList<Card> cardsHeld;
+	
+	/******************************************************************
+	 * Constructor method for Player class.
+	 * 
+	 * @param displayName Sets the player's name.
+	 * @param token Sets the player's chosen token.
+	 * @param wallet Sets the player's starting wallet value.
+	 *****************************************************************/
+	public Player(String displayName, String token, int wallet) {
+		super();
```

Comment:
Remove super(). Its auto generated by eclipse and shouldn't be there

Reply:
Doesn't it need to be there if we aren't using eclipse? But shouldn't it be super(name); and name be a param?

Reply:
When auto generating a constructor using eclipse, it generates that super() line a when the constructor is built. Player isn't the subclass of anything and therefore the super() call isn't necessary.  super(name) is in my PropertySquare Class because I called the Base class constructor of PropertySquare which has only one parameter which is the name of the board square.

Reply:
Oh duh, LOL. Sorry I was also working on a subclass of BoardSquare and got confused.